client/my-sites/sharing/connections/services/eventbrite.js

```diff
@@ -1,61 +1,48 @@
 /**
  * External dependencies
  */
-var last = require( 'lodash/last' );
+import { last } from 'lodash';
 
 /**
  * Internal dependencies
  */
-var connectionsList = require( 'lib/connections-list' )(),
-	sites = require( 'lib/sites-list' )();
+import connectionsList from 'lib/connections-list';
+import sites from 'lib/sites-list';
```

Comment:
Could we perhaps either connect this or update the functions to take in a site object instead of a siteId?